### PR TITLE
Include php 7.2 on travis

### PR DESCRIPTION
**Pull request by cordoval**
addresses https://github.com/liip/LiipMonitorBundle/issues/157

### CODE REVIEW

**Comment by kbond** [comment]
Looks good to me

**Comment by lsmith77** [comment]
I don't think this is needed. `1.1.0` should already be compatible with the current composer.json since we use `^1.0.2` and not `~1.0.2`

**Comment by kbond** [comment]
We should add 7.2 to the build matrix.

**Comment by cordoval** [comment]
you are right, theoretically it should work

**Comment by cordoval** [comment]
@lsmith77 let's see

**Comment by kbond** [comment]
had to restart some errored jobs but all passing

**Comment by cordoval** [comment]
\LGTM

**Comment by lsmith77** [comment]
awesome .. thx @cordoval

File: .travis.yml [hunk 1]
```diff
@@ -24,11 +25,13 @@ matrix:
...
     - php: 7.1
       env: SYMFONY_VERSION=4.0.*
+    - php: 7.2
+      env: SYMFONY_VERSION=4.0.*      
     - php: 5.6
       env: SYMFONY_VERSION=2.7.*;COMPOSER_FLAGS="--prefer-lowest"
 
 before_install:
-  - if [[ $TRAVIS_PHP_VERSION != '7.0' && $TRAVIS_PHP_VERSION != '7.1' ]]; then phpenv config-rm xdebug.ini; fi
+  - if [[ $TRAVIS_PHP_VERSION != '7.0' && $TRAVIS_PHP_VERSION != '7.1' && $TRAVIS_PHP_VERSION != '7.2' ]]; then phpenv config-rm xdebug.ini; fi
```
Comment (cordoval) [inline .travis.yml]: i honestly wonder what is the purpose of this line @lsmith77

Comment (lsmith77) [reply .travis.yml]: disabling the xdebug extension

Comment (cordoval) [reply .travis.yml]: but why for those versions

Comment (lsmith77) [reply .travis.yml]: mostly because php 7.0 is faster and more efficient so xdebug doesn't hurt so much .. IIRC

**Pull request merged**